### PR TITLE
fix: update docs on the storybook main page

### PR DESCRIPTION
**Pull request by burakukula**
Hey F36!

Just a small update on the package usage on the main page of react components storybook.

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx [hunk 1]
```diff
@@ -21,3 +21,43 @@ Design tokens can now be found on the [Forma 36 documentation website](https://f
...
+yarn add @contentful/forma-36-tokens
+```
+
+Or for NPM
+
+```bash
+npm i @contentful/forma-36-tokens
+```
+
+Import into your project
```
Comment (denkristoffer) [inline packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx:39]: Is it clear enough that all of these are needed?

**Review by denkristoffer — APPROVED** [review]

File: packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx (hunk 1 above)
Comment (burakukula) [reply packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx:39]: hmm, yeah, maybe you are right. Maybe we could say: Import all the following into your project. Or something like that?

Comment (burakukula) [reply packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx:39]: w8 I think I made a mistake here. ups.

File: packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx [hunk 2]
```diff
@@ -21,3 +21,29 @@ Design tokens can now be found on the [Forma 36 documentation website](https://f
...
+
+```jsx
+import { Input } from '@contentful/forma-36-react-components';
+```
+
+##### Import styles
+
+```js
+import '@contentful/forma-36-react-components/dist/styles.css';
+```
```
Comment (gui-santos) [inline packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx:49]: Wow! I didn't know we had to do this 👀 
Do we do that in the webapp?

**Review by gui-santos — APPROVED** [review]

File: packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx (hunk 2 above)
Comment (burakukula) [reply packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx:49]: yeah 🤯, we have it in the webpack config.

**Pull request merged**